pom.xml

```diff
@@ -103,6 +103,20 @@
     </organization>
 
     <properties>
+        <currentVersion>${project.version}</currentVersion>
+        <struts2.springPlatformVersion>5.3.18.RELEASE</struts2.springPlatformVersion>
+        <ognl.version>3.1.8</ognl.version>
+        <asm.version>3.3</asm.version>
+        <asm5.version>5.0.2</asm5.version>
+        <tiles.version>3.0.5</tiles.version>
+        <tiles-request.version>1.0.6</tiles-request.version>
+        <log4j2.version>2.5</log4j2.version>
```

Comment:
You have duplicated these properties and some of them have been reverted to very old versions. Also this changes nothing as the same properties are defined a few lines below which means they will overwrite your changes - that's why all the tests have passed.